buildpipeline/windows.groovy

```diff
@@ -80,15 +80,21 @@ simpleNode('Windows_NT','latest') {
                 def helixCreator = getUser()
 
                 // Target queues
-                def targetHelixQueues = ['Windows.10.Amd64.Open',
+                def targetHelixQueues = []
+                if (params.TGroup == 'netcoreapp')
+                {
+                    targetHelixQueues = ['Windows.10.Amd64.Open',
                                          'Windows.7.Amd64.Open',
                                          'Windows.81.Amd64.Open']
+                } else if (params.TGroup == 'uap') {
+                    targetHelixQueues = ['Windows.10.Amd64.ClientRS2.Open']
+                }
 
                 if (params.AGroup == 'x64') {
                     targetHelixQueues += ['Windows.10.Nano.Amd64.Open']
```

Comment:
I don't think we want this for uap. probably best to add this under the netcoreapp section.